src/pages/filter/ui/selected-group-items.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactElement } from 'react'
+import Checkbox from 'react-three-state-checkbox'
+import { observer } from 'mobx-react-lite'
+
+import { IVariantList } from '@declarations'
+import filterStore from '@store/filter'
+
+interface IProps {
+  chunks: any[]
+  currentPage: number
+  handleSelect: (index: number, checked: boolean) => void
+}
+
+export const SelectedGroupItems = observer(
+  ({ chunks, currentPage, handleSelect }: IProps): ReactElement => {
+    return (
+      <div className="mt-4">
+        {chunks[currentPage]?.map((element: IVariantList, index: number) => {
+          const { variant, isChecked } = element
+          const name = variant[0]
+          const amount = variant[1]
+
+          return (
+            <div
+              key={name}
+              className="flex items-center mb-3 text-14  leading-16px"
+            >
+              <Checkbox
+                checked={isChecked}
+                style={{ cursor: 'pointer' }}
+                indeterminate={filterStore.selectedGroupItem.name === name}
+                onChange={event => {
+                  handleSelect(index, event.target.checked)
+                }}
+              />
+
+              <span key={name} className="font-medium ml-2">
```

Comment:
Why do you need a key here?

Reply:
Apparently it was left from the previous implementation, but I forgot to remove.  I fixed